ovh/resource_cloud_project_kube_test.go

```diff
@@ -79,6 +79,24 @@ resource "ovh_cloud_project_kube" "cluster" {
 }
 `
 
+var testAccCloudProjectKubeUpdateLoadBalancersSubnetIdConfig = `
```

Comment:
to test the update, cluster must be attached to a subnet

this means to have a private network and 3 subnets:
- for node subnet
- for load balancer subnet
- to update the load balancer subnet

We can use TestAccCloudProjectKubeVRack with its configuration as a base to build the acceptance test

Reply:
Test updated, but not sure I'm doing it right